src/contracts/Repository/Values/ContentType/SearchResult.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Repository\Values\ContentType;
+
+use ArrayIterator;
+use Ibexa\Contracts\Core\Repository\Values\ValueObject;
+use IteratorAggregate;
+use Traversable;
+
+/**
+ * @implements \IteratorAggregate<int, \Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType>
+ */
+final class SearchResult extends ValueObject implements IteratorAggregate
+{
+    public int $totalCount = 0;
```

Comment:
```suggestion
    private int $totalCount = 0;
```

Could you plese add getTotalCount() getter ?